bower.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "adapt-contrib-hotgraphic",
-  "version": "2.1.1",
-  "framework": ">=2",
+  "version": "2.1.3",
```

Comment:
I think this will probably need to change to 3.0.0 since this is a major new bit of feature development - and we've said no more IE8/9/10 support for new features